docs/resources/google_compute_instances.md.erb

```diff
@@ -57,11 +57,12 @@ The following examples show how to use this InSpec audit resource.
 
 ## Filter Criteria
 
-This resource currently does not support any filter criteria; it will always fetch all instances in the zone.
+This resource supports the following filter criteria:  `instance_id` and `instance_name`. Either of these may be used with `where`, as a block or as a method.
```

Comment:
If you need it for a blog post, let's ship it, but please make a tech debt issue - people fiind `where` very confusing, and need examples.